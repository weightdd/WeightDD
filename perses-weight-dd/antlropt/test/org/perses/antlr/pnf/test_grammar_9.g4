grammar test_grammar_9;

start
    : ('A' | 'B') 'C'+ 'D'* ('E' | 'F')?
    ;