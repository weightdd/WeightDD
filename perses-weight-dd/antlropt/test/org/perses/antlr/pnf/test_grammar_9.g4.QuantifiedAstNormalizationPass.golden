// Auto-generated by Perses AST.
// DO NOT MODIFY.
grammar test_grammar_9;



start
    : ('A' | 'B') kleene_plus__start_1 kleene_star__start_2 optional__start_4
    ;

kleene_plus__start_1
    : 'C'+
    ;

kleene_star__start_2
    : 'D'*
    ;

aux_rule__start_3
    : 'E'
    | 'F'
    ;

optional__start_4
    : aux_rule__start_3?
    ;

